### PR TITLE
Remove constants and autoloader from the uninstall script

### PR DESCRIPTION
**Pull request by peterfabian**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR should solve two problems:
1. Constants are not loaded in uninstall.php script, so the check for WC_REMOVE_ALL_DATA would produce fatal error, resulting in an inability to remove WC
2. Cleanup for WC Admin has been simplified to use a basic include and direct call as we do with `WC_Install` class.

### How to test the changes in this Pull Request:

1. Try to uninstall WC, it should fail in master.
2. Try to uninstall with the patch applied, it should work.
3. Backup your db
4. Define `WC_REMOVE_ALL_DATA` in your `wp-config.php`
5. Remove WC, check that data is gone.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Fix - Removed constants and autoloader from the uninstall script.

### CODE REVIEW

**Comment by peterfabian** [comment]
cc @jeffstieler @ObliviousHarmony

**Comment by peterfabian** [comment]
This is also related to the problems raised in https://github.com/woocommerce/woocommerce/pull/25011#discussion_r375550886

File: uninstall.php [hunk 1]
```diff
@@ -27,20 +25,10 @@
...
-		require __DIR__ . '/src/Autoloader.php';
-
-		if ( \Automattic\WooCommerce\Autoloader::init() ) {
-			// Hook in WooCommerce Admin installation code.
-			// @todo an alternate way to do this is allow FeaturePlugin::has_satisfied_dependencies()
-			// to work during uninstall - (no WooCommerce instance).
-			\Automattic\WooCommerce\Admin\Install::drop_tables();
-		}
-	}
+if ( defined( 'WC_REMOVE_ALL_DATA' ) ) {
```
Comment (rodrigoprimo) [inline uninstall.php]: I think it is better to restore the original behavior and check if the constant is defined and is set to true:

https://github.com/woocommerce/woocommerce/blame/9c8a6d78d9b358aeace95b39b10c9fcde363efcb/uninstall.php#L28

**Review by rodrigoprimo — COMMENTED** [review]
Looks good to me. I just left one suggestion and then I think this is ready to be merged.

File: uninstall.php (hunk 1 above)
Comment (peterfabian) [reply uninstall.php]: Good idea, fixed in https://github.com/woocommerce/woocommerce/pull/25589/commits/42731d34e63b22f2f6384151859ca09c0316c07c

Comment (rodrigoprimo) [reply uninstall.php]: LGTM, thanks!

**Review by rodrigoprimo — APPROVED** [review]

**Comment by peterfabian** [comment]
also related to https://github.com/woocommerce/woocommerce/pull/25516

**Pull request merged**